apps/organisasjon-mottak-service/src/main/java/no/nav/registre/testnorge/organisasjonmottak/listener/OrganaisjonMottakListener.java

```diff
@@ -0,0 +1,56 @@
+package no.nav.registre.testnorge.organisasjonmottak.listener;
+
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.stereotype.Component;
+
+import no.nav.registre.testnorge.libs.avro.organiasjon.Ansatte;
+import no.nav.registre.testnorge.libs.avro.organiasjon.DetaljertNavn;
+import no.nav.registre.testnorge.libs.avro.organiasjon.Navn;
+import no.nav.registre.testnorge.libs.avro.organiasjon.Organiasjon;
+import no.nav.registre.testnorge.organisasjonmottak.service.OrganiasjonService;
+
+@Slf4j
+@Profile("prod")
+@Component
+@RequiredArgsConstructor
+public class OrganaisjonMottakListener {
+    private final OrganiasjonService organiasjonService;
+
+    @KafkaListener(topics = "tn-opprett-organiasjon-v1")
+    public void opprettOrganiasjon(@Payload Organiasjon organiasjon) {
```

Comment:
`public void opprettOrganisasjon(@Payload Organisasjon organisasjon)`
Samme for `opprettOrganisasjon` på de 3 metodene under 😄

Reply:
Alt for vanskilg å skrive organisasjon